Lib/http/server.py

```diff
@@ -713,6 +714,13 @@ def do_HEAD(self):
         if f:
             f.close()
 
+    def send_custom_response_headers(self):
```

Comment:
Should we simply have "send_response_headers"? or is there a need to add "custom_response_headers"? is there a plan for a future distinction between these two?